.github/actions/pr_updater/action.yml

```diff
@@ -0,0 +1,30 @@
+---
+# SPDX-FileCopyrightText: (C) 2025 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+name: 'Update Pull Requests'
+description: 'Update all pull requests to the head of their respective base branches'
+inputs:
+  github_token:
+    description: 'GitHub Token'
+    required: true
+runs:
+  using: 'composite'
+  steps:
+    - name: Authenticate with GitHub CLI
+      env:
+        github_token: ${{ inputs.github_token }}
+      shell: bash
+      run: echo "${github_token}" | gh auth login --with-token
+
+    - name: Update pull requests
+      shell: bash
+      run: |
+        mapfile -t PRs < <(gh pr list --json number --jq '.[].number')
+
+        # Iterate over each pull request and update it
+        for PR in "${PRs[@]}"; do
+          echo "Updating PR #$PR"
+          # Update the pull request branch with the latest changes from the base branch
+          gh pr update-branch "$PR" || echo "Failed to update PR #$PR"
+        done
```

Comment:
We should update only PRs with auto-merge enabled.

Reply:
done